.drone/drone.yml

```diff
@@ -70,9 +70,9 @@ steps:
     api_key:
       from_secret: GITHUB_TOKEN
     draft: true
-    files: dist/*
+    files: "dist/*"
```

Comment:
those changes are out of scope of the original intent but were generated with
```
$ jsonnet --version
Jsonnet commandline interpreter v0.16.0
$ make drone
drone jsonnet --source .drone/drone.jsonnet --target .drone/drone.yml --stream --format
```
so I committed them as well

Reply:
That's fine, probably `drone-cli` got an update in the meantime